internal/provider/resource_bgp_session_test.go

```diff
@@ -0,0 +1,36 @@
+package provider
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/PacketFabric/terraform-provider-packetfabric/internal/testutil"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccPBgpSessionRequiredFields(t *testing.T) {
+
+	testutil.SkipIfEnvNotSet(t)
+
+	bgpSessionResult := testutil.RHclBgpSession()
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			testutil.PreCheck(t, []string{})
+		},
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config:             bgpSessionResult.Hcl,
+				ExpectNonEmptyPlan: true,
```

Comment:
Why is this set to true?

Reply:
@pontovinte. As I understand reading [terraform documentation](https://developer.hashicorp.com/terraform/plugin/sdkv2/best-practices/testing), we should not set this value to true unless we expect a failure.